nearlib/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nearlib",
   "description": "Javascript library to interact with near blockchain",
-  "version": "0.4.0",
+  "version": "0.4.2",
```

Comment:
is increment by 2 intended?

Reply:
Yeah. The 0.4.1 didn't fully resolve the issue. It only addressed connect part, but there was still a bug with `getAccountId`. Not sure why there is no test to cover it. I'll probably add it before merging this.